mgmtsystem_audit/wizard/copy_verification_lines.py

```diff
@@ -19,36 +19,29 @@
 #
 ##############################################################################
 
-from openerp.osv import fields, orm
+from openerp import fields, models, api
 
 
-class copy_verification_lines(orm.TransientModel):
-    """
-    Copy Verification Lines
-    """
+class CopyVerificationLines(models.TransientModel):
+    """Copy Verification Lines."""
     _name = "copy.verification.lines"
     _description = "Copy Verification Lines"
-    _columns = {
-        'audit_src': fields.many2one('mgmtsystem.audit', 'Choose audit'),
-    }
 
-    def copy(self, cr, uid, ids, context=None):
-        # Copy verification lines from the chosen audit to the current one
-        if context is None:
-            context = {}
-
-        audit_proxy = self.pool.get(context.get('active_model'))
-        verification_line_proxy = self.pool.get('mgmtsystem.verification.line')
-        src_id = self.read(
-            cr, uid, ids, ['audit_src'], context=context)[0]['audit_src'][0]
+    audit_src = fields.Many2one('mgmtsystem.audit', 'Choose audit')
 
-        for line in audit_proxy.browse(
-                cr, uid, src_id, context=context).line_ids:
-            verification_line_proxy.create(cr, uid, {
+    @api.multi
+    def copy(self):
+        # Copy verification lines from the chosen audit to the current one
+        audit_proxy = self.env[self._context.get('active_model')]
+        verification_line_proxy = self.env['mgmtsystem.verification.line']
+        audit_id = self._context.get('active_id')
+        src_id = self.read(['audit_src'])[0]['audit_src'][0]
+        for line in audit_proxy.browse(src_id).line_ids:
+            verification_line_proxy.create({
                 'seq': line.seq,
                 'name': line.name,
-                'audit_id': context['active_id'],
+                'audit_id': audit_id,
                 'procedure_id': line.procedure_id.id,
                 'is_conformed': False,
-            }, context=context)
+            })
         return {'type': 'ir.actions.act_window_close'}
```

Comment:
@dreispt 
Please , here is the model of a wizard inside this module that i migrate, it is not working and i don't know what's wrong. Please i want you to check if you can figure out what is not working.